### DISCUSSION TITLE
Cannot apply for educational account due to the linked email to another account

### DISCUSSION THREAD

**Discussion by lanureeeh** [community/community#141646] [GitHub Education]
### Hi everyone,

I have lost an access to my previous account with the student GitHub, so I had to unlink it. And now, I want to get the student access for this account but I cannot make it, since it says that my email was already used on another account. Can I ask you to fix it?

**Comment by Sakib99-cyber** [comment]
Also i need to know about it.  Please, anyone, give a solution for the problem.

**Comment by davevad93** [comment]
Hi @lanureeeh ,

The issue is that the benefits of the `Student Developer Pack` are tied to the GitHub account and edu email that submitted the request. I don't know if there's a solution for this, but try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.
